src/util/string_utils.cpp

```diff
@@ -25,22 +26,18 @@ Author: Daniel Poetzl
 
 std::string strip_string(const std::string &s)
 {
-  std::string::size_type n=s.length();
+  auto pred=[](char c){ return std::isspace(c); };
 
-  // find first non-space char
-  unsigned i;
-  for(i=0; i<n; i++)
-  {
-    if(!std::isspace(s[i]))
-      break;
-  }
-  if(i==n)
+  std::string::const_iterator left
+    =std::find_if_not(s.begin(), s.end(), pred);
```

Comment:
Nit picking (applies here and for the other case of std::find_if_not): place the "=" at the end of the previous line, or maybe even use `auto left=...` and put it all on a single line.